tests/integration/pilot/minimal/testdata/crds.yaml

```diff
@@ -0,0 +1,396 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
@howardjohn this file (to me) appears to be misnamed. Would you consider `knative-crds.yaml`?